server/js/sockets.js

```diff
@@ -194,7 +153,47 @@ sockets.init = function socketInit(server) {
           );
         }
       } else {
-        console.error(`Unhandled topic - ${topic}`);
+        switch (topic) {
+          case DAS.start:
+            socket.emit('start');
+            break;
+          case DAS.stop:
+            socket.emit('stop');
+            break;
+          case DAS.data:
+            mqttDataTopicHandler(socket, payloadString);
+            if (sendToPublicMQTTBroker()) {
+              PUBLIC_MQTT_CLIENT.publish(DAS.data, payloadString);
+            }
+            break;
+
+          case BOOST.predicted_max_speed:
+            socket.emit('power-model-running');
+            socket.emit(
+              'power-model-max-speed',
+              queryStringToJson(payloadString),
+            );
+            break;
+          case BOOST.recommended_sp:
+            socket.emit('power-model-running');
+            socket.emit(
+              'power-model-recommended-SP',
+              queryStringToJson(payloadString),
+            );
+            break;
+          case 'power_model/plan_generated':
```

Comment:
Same as before would be nice to note that this should be removed when `BOOST.generate` is implemented.